### DISCUSSION TITLE
Dependency Dashboard header no longer always on top

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#32103] [Request Help] [answered]
### How are you running Renovate?

None

### Please tell us more about your question or problem

According to documentation, the `dependencyDashboardHeader` is "Any text added here will be placed first in the Dependency Dashboard issue body."`. This is not true anymore, since #31129 was added. The header is now under the H2 for "Config(uration?) Migration Needed":

```
## Config Migration Needed

<!-- config-migration-pr-info --> See Config Migration MR: #93.

This issue lists Renovate updates and detected dependencies. Read the [Dependency Dashboard](https://docs.renovatebot.com/key-concepts/dashboard/) docs to learn more.

> ⚠ **Warning**
> 
These dependencies are deprecated:

| Datasource | Name | Replacement MR? |
|------------|------|--------------|
| dockerfile | `node` | ![Available](https://img.shields.io/badge/available-green?style=flat-square) |
```

In my opinion, the section that @RahulGautamSingh added to `ensureDependencyDashboard` for it should be shifted down to after the header.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Converted to issue:

- https://github.com/renovatebot/renovate/issues/32104

---

Edit by @HonkingGoose:

And this PR will move the header back to the top:

- #32160
